### ISSUE TITLE
Ability to hide certain characters (i.e. animals, allies) from the map view

### ISSUE THREAD

**Issue by Tum1370** [Dark-Basic-Software-Limited/GameGuruRepo#4456]
As the title says, The "map:character" shows all mammals on the map, as well as showing the characters.

**Comment by Necrym59** [comment]
No way to discern mamals apart from other mammals yet as far as i know

**Comment by davetgc** [comment]
Agreed, not a bug, but not really an expected feature unless they were a quest objective.
